region/lowernorfair/east.json

```diff
@@ -842,9 +854,28 @@
           "to": [
             {
               "id": 1,
-              "requires": [ "canSwim" ],
+              "requires": [
+                {"or": [
+                  "heatProof",
+                  "canHeatRun"
+                ]},
+                {"and": [
```

Comment:
This `and` (and the next one) are already within an implicit `and`. Were they intended to be two options in an `or`?

Reply:
They're meant to be handling gravity vs suitless but I forgot the "or" and also for some reason the gravity one got no heatframes.

Reply:
Makes sense. Should be fine once fixed then.